cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -327,18 +328,25 @@ The following commands are only executable against the `system` database:
+| <<administration-security-administration-dbms-privileges-user-management, DROP USER>>         | DBMS                  | Determines whether the user can delete users.
 | <<administration-security-administration-database-indexes, INDEX MANAGEMENT>>                 | Schema                | Determines whether a user is allowed to create and drop indexes on a specific database.
 | <<administration-security-subgraph-match, MATCH>>                                             | GRAPH                 | Determines whether the properties of an element (node, relationship or both) can be read and the element can be found and traversed while executing queries on the specified graph.
 | <<administration-security-administration-database-tokens, NAME MANAGEMENT>>                   | Schema                | Determines whether a user is allowed to create new labels, types and property names on a specific database.
 | <<administration-security-subgraph-read, READ>>                                               | GRAPH                 | Determines whether the properties of an element (node, relationship or both) can be read while executing queries on the specified graph.
 | <<administration-security-administration-dbms-privileges-role-management, REMOVE ROLE>>       | DBMS                  | Determines whether the user can revoke roles.
 | <<administration-security-administration-dbms-privileges-role-management, ROLE MANAGEMENT>>   | DBMS                  | Determines whether the user can create, drop, grant, revoke and show roles.
 | <<administration-security-administration-dbms-privileges-role-management, SHOW ROLE>>         | DBMS                  | Determines whether the user can get information about existing and assigned roles.
+| <<administration-security-administration-database-transaction, SHOW TRANSACTION>>             | Database              | Determines whether a user is allowed to list transactions and queries.
+| <<administration-security-administration-dbms-privileges-user-management, SHOW USER>>         | DBMS                  | Determines whether the user can get information about existing users.
 | <<administration-security-administration-database-startstop, START>>                          | Database              | Determines whether a user can start up a specific database.
 | <<administration-security-administration-database-startstop, STOP>>                           | Database              | Determines whether a user can stop a specific running database.
+| <<administration-security-administration-database-transaction, TERMINATE TRANSACTION>>        | Database              | Determines whether a user is allowed to kill transactions and queries.
+| <<administration-security-administration-database-transaction, TRANSACTION MANAGEMENT>>       | Database              | Determines whether a user is allowed to list and kill transactions and queries.
```

Comment:
we try to avoid using emotive terminology such as `kill`.  A replacement such as `abort` is also problematic.

Without a styleguide (coming in 2020 🤞 ), I'm not sure how consistent we are with this. I would recommend rewording these sentences with something like:

 >Determines whether a user is allowed to end running transactions and queries.

Reply:
That could work, I chose the word since the procedures used for it is called killed but have no issue with using end :)

Reply:
ok cool thanks - shall I push the amendments?

Reply:
done 👍